src/test/scala/chiselTests/Module.scala

```diff
@@ -8,6 +8,8 @@ import chisel3.stage.{ChiselGeneratorAnnotation, ChiselStage, NoRunFirrtlCompile
 import firrtl.annotations.NoTargetAnnotation
 import firrtl.options.Unserializable
 
+import scala.io.Source
```

Comment:
I suggest os-lib here.

Reply:
```scala.io.Source``` is so super easy to use and a Scala standard library. I do not see any benefit from a third-party library in this case.

Reply:
the reason not using Scala.io is because os-lib provides a typesafe way to play with system path for different operation system, while Scala native io is just a wrapper to Java io, which takes String as parameter(not type safe), i personally think this is a bad pattern, and I’m trying to push this replacement in the chisel environment.
Since os-lib is introduced as a dependency, so I think we don’t need to care about is third party or not, we can just align to it since we already depends on that.
We can leave this discussion in dev meeting today.